python/pyarrow/_rados.pxd

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I wonder if we need a brand new module here or not.  For example, it could be in the dataset module and then it could be an error at use time if someone tries to use it.  Currently the parquet file format and csv file format live in the dataset module (even though there is already a parquet module and csv module).  However, those two are generally guaranteed to be installed.  I'm not sure which pattern to follow for rados.